### PR TITLE
ref(ts) Convert Organization stats to typescript

### PR DESCRIPTION
**Pull request by markstory**
Get the TS conversion done so that I can replace the chart.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationStats/organizationStatsDetails.tsx [hunk 1]
```diff
@@ -142,17 +145,13 @@ class OrganizationStats extends React.Component {
...
         </Panel>
-        {pageLinks && <Pagination pageLinks={pageLinks} {...this.props} />}
+        {this.props.pageLinks && <Pagination {...this.props} />}
       </div>
     );
   }
 
   render() {
-    return (
-      <React.Fragment>
```
Comment (scttcper) [inline src/sentry/static/sentry/app/views/organizationStats/organizationStatsDetails.tsx:152]: nice

**Comment by markstory** [comment]
I forgot to update one file in this view. I'll do that now.

File: src/sentry/static/sentry/app/views/organizationStats/index.tsx [hunk 2]
```diff
@@ -210,24 +241,26 @@ class OrganizationStatsContainer extends React.Component {
...
-    const projectTotals = [];
-    $.each(rawProjectData.received, (projectId, data) => {
+    const projectTotals: ProjectTotal[] = [];
+    Object.keys(rawProjectData.received).forEach(projectId => {
+      const data = rawProjectData.received[projectId];
       let pReceived = 0;
       let pRejected = 0;
       let pBlacklisted = 0;
-      $.each(data, (idx, point) => {
+      data.forEach((point, idx) => {
```
Comment (scttcper) [inline src/sentry/static/sentry/app/views/organizationStats/index.tsx:263]: yes!!

**Review by scttcper — APPROVED** [review]

**Review by priscilawebdev — APPROVED** [review]
Good Job 👏

**Comment by markstory** [comment]
Screen Shot 2020-10-27 at 12 37 40 PM

@matejminar Fixed the centering. Thanks for spotting that.

**Pull request merged**